create3_lidar/README.md

```diff
@@ -42,18 +42,25 @@ Note that all STLs of brackets referenced in this example are found in our [crea
 
 ### SBC Setup
 
-On the SBC, clone and build this, and source the setup shell scripts.
+On the SBC, clone and build the [create3_examples repository](https://github.com/iRobotEducation/create3_examples). 
 
-Start the sensors launch script, which includes the LIDAR driver and transform from the laser scan to the robot:
+Source the setup shell scripts in **EVERY** terminal you open:
 
 ```bash
-ros2 launch create3_lidar sensors_launch.py
+source ~/create3_examples_ws/install/local_setup.sh 
 ```
 
-In a separate terminal run slam toolbox:
+Run the sensors launch script, which includes the LIDAR driver and transform from the laser scan to the robot. Replace `ROBOTNAMESPACE` with the namespace of your robot. If you are not using a namespace, you may delete `namespace:='ROBOTNAMESPACE'` from the command:
 
 ```bash
-ros2 launch create3_lidar slam_toolbox_launch.py
+ros2 launch create3_lidar sensors_launch.py namespace:='ROBOTNAMESPACE'
```

Comment:
minor: rather than showing the example with a namespace and having the description above saying to replace/remove it, I would rather do something like this

```
Run the sensors launch script, which includes the LIDAR driver and transform from the laser scan to the robot.

ros2 launch create3_lidar slam_toolbox_launch.py


If your robot is using a namespace, you should add namespace:='ROBOTNAMESPACE' to the previous command, where ROBOTNAMESPACE is the namespace of your robot
```

I think that it's easier to read / understand